lib/matplotlib/axes/_base.py

```diff
@@ -2222,7 +2222,7 @@ def add_artist(self, a):
 
     def add_child_axes(self, ax):
         """
-        Add an `.AxesBase` to the Axes' children; return the child Axes.
+        Add an ``AxesBase`` to the Axes' children; return the child Axes.
```

Comment:
```suggestion
        Add an Axes to this Axes' children; return the child Axes.
```
The exact type does not matter here.